ibm/mas_devops/roles/kmodels/vars/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+# vars file for kmodels
+pullSecretName: "ibm-entitlement"
+# controller
+controllerTag: 1.0.0-pre.10xdev
+controller: "aibroker/controller:{{ controllerTag }}"
+# store
+storeTag: 1.0.0-pre.10xdev
+store: "aibroker/store:{{ storeTag }}"
+# watcher
+watcherTag: 1.0.0-pre.10xdev
```

Comment:
We would need to remove all the pre releases in this file

Reply:
changed to release versions